lib/classes/session/manager.php

```diff
@@ -1002,7 +1002,7 @@ public static function gc() {
             $rs->close();
 
             // Delete expired sessions for guest user account, give them larger timeout, there is no security risk here.
-            $params = array('purgebefore' => (time() - ($maxlifetime * 5)), 'guestid'=>$CFG->siteguest);
+            $params = array('purgebefore' => (time() - $maxlifetime), 'guestid'=>$CFG->siteguest);
```

Comment:
We prefer the short syntax of array.

```suggestion
            $params = ['purgebefore' => (time() - $maxlifetime), 'guestid'=>$CFG->siteguest];
```

Reply:
Added changes